src/Core/CrestApps.OrchardCore.Subscriptions.Core/SubscriptionPermissions.cs

```diff
@@ -8,5 +8,7 @@ public static class SubscriptionPermissions
 
     public static readonly Permission ManageSubscriptions = new("ManageSubscriptions", "Manage subscriptions");
 
-    public static readonly Permission ManageOwnSubscriptions = new("ManageSubscriptions", "Manage subscriptions", [ManageSubscriptions]);
+    public static readonly Permission ManageOwnSubscriptions = new("ManageOwnSubscriptions", "Manage Own subscriptions", [ManageSubscriptions]);
+
+    public static readonly Permission AccessSubscriberDashboard = new("AccessSubscriberDashboard", "Access Subscriber Dashboard");
```

Comment:
This would only be applicable on the backend. The user automatically get access to the user's dashboard with no permission.